docs/database_schema_diagrams/source/entity_network_details.json

```diff
@@ -1,6 +1,6 @@
 {
   "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities, that is, beyond alias, collection, edit, link, subscription, rating, tag.",
+  "_comment": "It shows tables unique to some entities, that is, beyond alias, collection, edit, relationship, subscription, rating, tag.",
```

Comment:
The actual table is `link`, but not sure if that matters.

Edit: I see you're moving away from table names on the next commit, so assuming that's a good idea then it does not matter :)

Reply:
Correct.